### PR TITLE
add(types): stage discovery privacy fields

### PR DESCRIPTION
**Pull request by rigormorrtiss**
Re﻿ference: https://github.com/discord/discord-api-docs/pull/2971

### CODE REVIEW

File: src/types/channels/privacy_level.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// TODO: add resource link
+export enum PrivacyLevel {
+  /** The Stage instance is visible publicly, such as on Stage discovery */
+  Public,
```
Comment (itohatweb) [inline src/types/channels/privacy_level.ts:4]: enums start at 0 but discord at 1
```suggestion
  Public = 1,
```

**Review by itohatweb — CHANGES_REQUESTED** [review]

**Comment by itohatweb** [comment]
Please resolve the conflicts

**Comment by Skillz4Killz** [comment]
Need unit tests

**Comment by itohatweb** [comment]
Did you test the changes localy and can you confirm that they work?

**Comment by rigormorrtiss** [comment]
> Did you test the changes localy and can you confirm that they work?

Yes

**Comment by itohatweb** [comment]
@Skillz4Killz I will merge it now since it is impossible to do tests rn because discord does not allow bots to create conmunity guilds.

**Review by itohatweb — APPROVED** [review]

**Pull request merged**